From 0d16578d750a504ed6cddd0669d26fd5ab3fd490 Mon Sep 17 00:00:00 2001
From: fang hui <b31070@freescale.com>
Date: Fri, 31 Jul 2015 11:09:39 +0800
Subject: [PATCH] MA-6918 - [Android_MX6SX_SD] Camera: Video recorded by on
 board camera

OV5640 is only 4fps. 100%

The csi clock is only 8M, should be 24M.

In ov5640_probe, The csi clock should not enabled before set it's rate.
But the clock is enabled in vadc_probe.

Schedule the sequence of vadc and ov5640 in Makefile, so the ov5640 is
loaded before vadc.

Signed-off-by: fang hui <b31070@freescale.com>
---
 drivers/media/platform/mxc/subdev/Makefile | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/drivers/media/platform/mxc/subdev/Makefile b/drivers/media/platform/mxc/subdev/Makefile
index 25b0d64..ff6e2bc 100644
--- a/drivers/media/platform/mxc/subdev/Makefile
+++ b/drivers/media/platform/mxc/subdev/Makefile
@@ -1,7 +1,6 @@
 #Makefile for mxc csi driver
 
 obj-$(CONFIG_VIDEO_MXC_CSI_CAMERA)	+= mx6s_capture.o
-obj-$(CONFIG_MXC_VADC) += mxc_vadc.o
 obj-$(CONFIG_MXC_MIPI_CSI) += mxc_mipi_csi.o
 
 ov5640_camera-objs := ov5640.o
@@ -12,3 +11,5 @@ obj-$(CONFIG_MXC_CAMERA_OV5640_MIPI) += ov5640_camera_mipi.o
 
 ov5647_camera_mipi-objs := ov5647_mipi.o
 obj-$(CONFIG_MXC_CAMERA_OV5647_MIPI) += ov5647_camera_mipi.o
+
+obj-$(CONFIG_MXC_VADC) += mxc_vadc.o
-- 
1.8.0

